### ISSUE TITLE
Button With Dropdown aria-expanded missing on initial render

### ISSUE THREAD

**Issue by GCHQ-Developer-299** [mi6/ic-ui-kit#1519]
## Summary of the bug 
Dropdown buttons don't read that they're expandable on initial render.

## 🪜 How to reproduce 

1. Load a page with a Dropdown button variant, with Voiceover active.
2. Note that it doesn't read whether it's expanded or collapsed. 
3. Open the dropdown, close the dropdown. 
4. Note that the button now reads that it's collapsed. 

Under the hood this is because the <button> element inside <ic-button> doesn't have aria-expanded=false set on initial render, only after being interacted with do we apply it. 

## 📸 Screenshots or code 
Check out the [Button Dropdown Example story](https://mi6.github.io/ic-ui-kit/branches/develop/web-components/?path=/story/web-components-button--dropdown-example)
